### ISSUE TITLE
tests 

### ISSUE THREAD

**Issue by walaamedhat** [FACG5/flawless-todo-list#34]
You made 5 tests for 5 functions.
But if you think more you will make 5 tests for one function.

for example :
` test('AddToDoTest', function(t) {
    var actual= todoFunctions.addTodo( [],{
      id:0,
      description: "",
      done: false,
    });

    var expected =[]

    t.deepEqual(actual, expected,"should return sleep");
    t.end();
  });`

**Comment by ali-7** [comment]
yes, but more thinking = more time :smiley: and we had no time as u know.
and thanks a lot for ur help :smile:

**Comment by walaamedhat** [comment]
You will learn to think more in a short time don't worry but that needs more effort from you.
You are welcome Ali, I'm always there to help you.  
:smile:
